### ISSUE TITLE
DurationField support

### ISSUE THREAD

**Issue by JerryMartinsonLeviti** [strawberry-graphql/strawberry-graphql-django#233]
## Feature Request Type

- [x] Alteration (enhancement/optimization) of existing feature(s)

## Description

I use DurationFields a fair amount in Django and I noticed that they don't seem to be supported.  Digging through the code a bit, I think it could be added by changing `.../strawberry_django/fields/types.py` in the following way:

Adding `fields.DurationField: datetime.timedelta` to the `field_type_map` `dict` between line 70 & 71.

I'm a total newbie to any contributory activity, so I'm not sure if there's a better way to submit this.

**Comment by bellini666** [comment]
Yes, it is basically that. But note that this will require a new scalar too, and I think that scalar should go do https://github.com/strawberry-graphql/strawberry/blob/main/strawberry/schema/types/base_scalars.py

Basically a scalar named `Timedelta` that serializes to `.total_seconds()` and parsers the seconds as a `timedelta` object

Then you just need to map the `DurationField` to `datetime.timedelta` and it is done
